### PR TITLE
Fix alpha bugs

### LINKED ISSUE #222: Rental end date should be later than rental start date
for radd and redit.

### LINKED ISSUE #223: Autocomplete should include all commands

### LINKED ISSUE #224: Help message displaying is still AB3

### LINKED ISSUE #232: Unspecified value in rental information should display as dash for better understanding

### LINKED ISSUE #233: radd and redit should accept parameters with empty string
User can provide empty string for parameters now, even though it is optional to do so as it will have the same behaviour if the parameters are not provided at all.
redit now allow user to set the value back to "empty" by using parameters with empty value,

### LINKED ISSUE #234: Add more tests for changes by alpha bug
Mainly for classes in RentalInformation, AddRentalCommandParser, EditRentalCommand, EditRentalCommandParser.

### PR DESCRIPTION
**Pull request by gohsl99**
Fixes #222, Fixes #223, Fixes #224, Fixes #232, Fixes #233, Fixes #234.

### CODE REVIEW

File: src/main/java/seedu/address/commons/util/RentalUtil.java [hunk 1]
```diff
@@ -67,7 +68,7 @@ public static ArrayList<String> convertStringToCustomerArrayList(String str) {
      * @param secondList the second {@code CustomerList} to compare.
      * @return {@code true} if both customer lists contain the same set of customers, {@code false} otherwise.
      */
-    public static boolean isCustomerListSame(CustomerList firstList, CustomerList secondList) {
+    public static boolean areCustomerListsSame(CustomerList firstList, CustomerList secondList) {
```
Comment (winstonlimjy) [inline src/main/java/seedu/address/commons/util/RentalUtil.java:71]: Nice change of naming

File: src/test/java/seedu/address/logic/parser/AddRentalCommandParserTest.java [hunk 2]
```diff
@@ -67,10 +70,17 @@ public void parse_allFieldsPresent_success() {
         assertParseSuccess(parser, PREAMBLE_WHITESPACE + " 1" + ADDRESS_DESC_ONE + RENTAL_START_DATE_DESC_ONE
                 + RENTAL_END_DATE_DESC_ONE + RENT_DUE_DATE_DESC_ONE + MONTHLY_RENT_DESC_ONE + DEPOSIT_DESC_ONE
                 + CUSTOMER_LIST_DESC_ONE, new AddRentalCommand(INDEX_FIRST_PERSON, expectedRental));
+
```
Comment (winstonlimjy) [inline src/test/java/seedu/address/logic/parser/AddRentalCommandParserTest.java:73]: Nice to see you updated the tests to fit the new requirements. goodjob

File: src/main/java/seedu/address/logic/commands/EditRentalCommand.java [hunk 3]
```diff
@@ -69,7 +70,8 @@ public class EditRentalCommand extends Command {
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_RENTAL = "This client already exists in the address book.";
     public static final String MESSAGE_DUPLICATE_RENTAL_INFORMATION =
-            "The editing rental information already exists in the address book";
+            "The editing rental information already exists in the address book.";
+    public static final String MESSAGE_INVALID_RENTAL_DATES = "Rental end date should be after rental start date.";
```
Comment (winstonlimjy) [inline src/main/java/seedu/address/logic/commands/EditRentalCommand.java:74]: Perhaps DATE is sufficient. Nothing serious

Comment (gohsl99) [reply src/main/java/seedu/address/logic/commands/EditRentalCommand.java:74]: There is also rent due date, so the name is to differentiate them.

**Pull request merged**